src/large_pages/node_text_start.S

```diff
@@ -0,0 +1,5 @@
+.text
+.align 0x2000
```

Comment:
Why 8k?

Reply:
Good question, actually. I'm not sure we need it, because we end up snipping the front and the back off the .text if it doesn't align at 2MiB, anyway, leaving the snipped-off bits on 4KiB pages.